dev-perl/DateTime-Format-Human-Duration/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>c.affolter@stepping-stone.ch</email>
+		<name>Christian Affolter</name>
+		<description>Maintainer</description>
+	</maintainer>
+	<maintainer type="person">
+		<email>dev-zero@gentoo.org</email>
+		<description>Proxy maintainer</description>
```

Comment:
If we add the proxy-maint block with dev-zero's permission, you don't need the description tags on yourself and him.